File: back/src/main/java/com/integradora/back/controller/orden/dto/OrdenResponseDTO.java
package com.integradora.back.controller.orden.dto;

import com.integradora.back.controller.detalleorden.dto.DetalleOrdenDTO;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;

@Data
@AllArgsConstructor
public class OrdenResponseDTO {

    private Long id;
    private String estado;
    private Integer mesaNumero;

    private List<DetalleOrdenDTO> items;

    private BigDecimal subtotal;
    private BigDecimal total;

    private LocalDateTime fechaCreacion;
}

File: mobile/app/src/main/java/proyecto/personal/proyectointegradorii/data/remote/dto/orden/OrdenResponseDTO.kt
package proyecto.personal.proyectointegradorii.data.remote.dto.orden

import proyecto.personal.proyectointegradorii.data.remote.dto.detalleorden.DetalleOrdenDTO

data class OrdenResponseDTO(
    val id: Long,
    val estado: String,
    val mesaNumero: Int?,
    val detalles: List<DetalleOrdenDTO>,
    val subtotal: Double,
    val montoDescuento: Double,
    val codigoPromoAplicado: String?,
    val total: Double,
    val fechaCreacion: String?
)